example/example/tests/test_views.py

```diff
@@ -124,4 +120,6 @@ def test_tracking_features(self, api_client, astrosat_settings):
         assert status.is_success(response.status_code)
         assert DatabaseLogRecord.objects.count() == 2
         assert DatabaseLogTag.objects.count() == 1
-        assert content == {"detail": "Log Created"}
+
+        for input_data, output_data in zip(log_data, content):
```

Comment:
Here is where you can re-order the content as mentioned above:

```
for input_data, output_data in zip(log_data, sorted(content, key=lambda x: x["id"])):
```

Reply:
I did it slightly differently and came across it by accident, so maybe not the best, but simpler to read, I will do whatever you prefer:

```python
    content.reverse()
    for input_data, output_data in zip(log_data, content):
        assert json.dumps(input_data['content']) == output_data['message']
```

Reply:
So I don't think that using `reverse` is guaranteed to work.  The order that the serializer returns data is undefined; reversing an unknown order still produces an unknown order.  Better to explicitly match the order that you know the records were created (in this instance it is the order of `log_data` will will be mirrored in the `ids` (since they're incrementing integers) of the serialized objects).